website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+
+1. What is the dbt semantic layer?
+2. How do I use it?
+3. What is publicly available now?
+4. What is still in development?
+
+With that, lets get into it!
+
+<!--truncate-->
+
+> Some of you might be confused - you’ve seen us talk about the Metrics Layer in the past and now we’re talking about the Semantic Layer. Well, what was previously called the Metrics Layer has been re-branded to the “Semantic Layer” in order to better reflect our long term plans. Today’s functionality focus largely on metrics but tomorrow’s will expand to be a programmatic interface to the DAG, to better understand entities and what models represent, and much more. Drew’s post about the future of the Semantic Layer explains this far better.
+
+## What is the dbt Semantic Layer?
+
+The dbt semantic layer is a combination of several components - some of these are part of `dbt-core`, some part of `dbt Cloud` (our SaaS offering), and some are net new functionality. They all [combine together like Voltron](https://www.youtube.com/watch?v=5rPSLQxMT8w) to create a single experience through which business users can query data in a context that is familiar to them — metrics. And the best part is that they can do it in systems they are already comfortable using.
+
+***What will this look like for my data consumers and business stakeholders?*** 
+
+Ultimately, this looks like people being able to interact with trusted datasets in the tools that they are comfortable with (and eventually new tools designed specifically around metrics). 
+
+An example that we’ve found helpful is [ARR](https://www.zuora.com/billing-topics/annual-recurring-revenue/#:~:text=Annual%20Recurring%20Revenue%2C%20or%20ARR,for%20a%20single%20calendar%20year). A business-critical metric to SaaS companies, ARR can be a tricky calculation to keep consistent across all of the tools used in the business. With the semantic layer, this definition would live in dbt and the logic to create the dataset for that metric would be consistent across all different consuming experiences. Best of all, definition changes would get reflected in downstream tools, so you no longer need to manually search and update every downstream dependency. Callum of 3 years ago is jumping with joy.
+
+***That’s good and all, but what does this look like for practitioners to use?*** 
+
+We’re building it with the following components[^1]:
+
+**Available Today**
+
+- `**[metric` node in `dbt-core` :](https://docs.getdbt.com/docs/building-a-dbt-project/metrics)** Similar to `models` or `sources` , this is a specific node type in `dbt-core`. It is the definition of a time-series aggregation over a table that supports 0+ dimensions. The resulting metadata is stored in the `manifest` just like `models` and referenced in the DAG.
```

Comment:
Something is up with the markedown formatting here think you might want to delete a space at the front of the first two entries here

Reply:
updated